src/index.jsx

```diff
@@ -2,7 +2,64 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { App } from './App';
 
+export const numbers1 = {
+  a: 2,
+  b: 3,
+};
+
+export const numbers2 = {
+  a: -5,
+  b: 5,
+};
+
+export const numbers3 = {
+  a: 10,
+  b: 0,
+};
+
+export const numbers4 = {
+  a: 0,
+  b: 5,
+};
+
+export const numbers5 = {
+  a: 0,
+  b: 0,
+};
+
 ReactDOM.render(
-  <App />,
+  <div>
+    <App {...numbers1} />
+    <App {...numbers2} />
+    <App {...numbers3} />
+    <App {...numbers4} />
+    <App {...numbers5} />
+  </div>,
```

Comment:
- use React.Fragment instead of div here
- you should write your code inside App.jsx and use Sum.jsx inside of it